import/repos.json

```diff
@@ -81,8 +81,8 @@
     "repo": "https://github.com/EventStore/EventStoreDB-Client-Java",
     "branches": [
       {
-        "version": "1.0.0",
-        "name": "trunk"
+        "version": "2.0.0",
+        "name": "persistent_subscriptions_samples"
```

Comment:
**TODO:** Update it once https://github.com/EventStore/EventStoreDB-Client-Java/pull/133 is merged.